python-3.13.yaml

```diff
@@ -1,7 +1,7 @@
 package:
   name: python-3.13
   version: "3.13.9"
-  epoch: 0
+  epoch: 1
```

Comment:
ha, I didn't actually change 3.13 :facepalm: 

if I pick "Delete File" from the "..." menu on the right, will it delete the *change* or the entire file? I don't trust it to do the right thing